github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```
@@ -538,6 +538,20 @@ presets:
   - name: kubevirtci-cred
     mountPath: /etc/kubevirtci-cred
     readOnly: true
+- labels:
+    preset-github-credentials: "true"
+  env:
+  - name: GIT_ASKPASS
+    value: ../project-infra/hack/git-askpass.sh
+  - name: GITHUB_TOKEN
+    value: "/etc/github/oauth"
+  volumes:
+  - name: github-token
+    secret:
+      secretName: oauth-token
```

Maybe we can set user readonly, but not sure whether that would work for every job?

```suggestion
      defaultMode: 0400
      secretName: oauth-token
```

---

Tested a couple of jobs and they succeeded with the above suggested permissions.